ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -51,26 +96,30 @@ class SizeComparisonViewController: UIViewController {
     }
 
     private func removeNodeIfExistAlready() {
-        if let nodeExists = sceneView.scene.rootNode.childNode(withName: currentCubeName, recursively: true) {
+        if let nodeExists = sceneView.scene.rootNode.childNode(withName: currentCube.filename, recursively: true) {
             nodeExists.removeFromParentNode()
         }
 
-        if let nodeExists = sceneView.scene.rootNode.childNode(withName: currentTextName, recursively: true) {
+        if let nodeExists = sceneView.scene.rootNode.childNode(withName: currentCube.textFilename, recursively: true) {
             nodeExists.removeFromParentNode()
         }
     }
 
     @IBAction func segmentHasBeenChanged(_ sender: UISegmentedControl) {
+        
+        //If the user selects ar measures, `realWorldMeasuresSegmentedControl` should be deselected
+        //otherwise `ARMeasuresSegmentedControl` should be deselected.
+        (sender == self.ARMeasuresSegmentedControl ? self.realWorldMeasuresSegmentedControl : self.ARMeasuresSegmentedControl).selectedSegmentIndex = UISegmentedControlNoSegment
```

Comment:
refactor this. Switch case ? over those two segmenteControl. Or if/else construct. Then the assignment.